DuckDuckGo/PrivacyProtectionEncryptionDetailController.swift

```diff
@@ -82,10 +82,12 @@ class PrivacyProtectionEncryptionDetailController: UIViewController {
         unencryptedLabel.isHidden = true
         mixedContentLabel.isHidden = true
```

Comment:
Would the various overlapping encryption labels in the storyboard that are shown / hidden in code be better as a single label that gets set to the correct text according to state?
• encryptedLabel
• unencryptedLabel
• mixedContentLabel

Reply:
Probably, it's a nightmare to look at it interface builder.  There's a similar situation in the privacy dashboard header, but that's because there's images embedded in the text. 

There's going to be a bit of UI work soon so I'll create a task for this as part of that.